feature-utils/poly-look/storybook/.storybook/lit/server.mjs

```diff
@@ -2,8 +2,10 @@ import { storybookPlugin } from "@web/dev-server-storybook";
 import baseConfig from "../../web-dev-server.config.mjs";
 import { fromRollup } from "@web/dev-server-rollup";
 import svg from "rollup-plugin-svg";
+import { nodeResolve } from "@web/dev-server-rollup";
```

Comment:
just add this above, besides `fromRollup`

Reply:
Done